pkg/csi/service/common/commonco/coagnostic.go

```diff
@@ -36,6 +37,10 @@ var ContainerOrchestratorUtility COCommonInterface
 type COCommonInterface interface {
 	// Check if feature state switch is enabled for the given feature indicated by featureName
 	IsFSSEnabled(ctx context.Context, featureName string) bool
+	// Check if the passed volume can be fake attached and mark it as fake attached.
+	IsFakeAttachAllowed(ctx context.Context, volumeID string, volumeManager cnsvolume.Manager) (bool, error)
```

Comment:
Can we avoid taking volumeManager as an argument?

Reply:
I too thought so that it shouldn't be an argument to an interface method.
But if we avoid passing `volumeManager` as an argument here, we will need to initialize it in k8sOrchestrator instance and have pass it to argument in `GetContainerOrchestratorInterface()` method, which doesn't feel neat either IMO.